tools/pr_comments.py

```diff
@@ -0,0 +1,35 @@
+# This file is part of the EESSI build-and-deploy bot,
+# see https://github.com/EESSI/eessi-bot-software-layer
+#
+# The bot helps with requests to add software installations to the
+# EESSI software layer, see https://github.com/EESSI/software-layer
+#
+# author: Kenneth Hoste (@boegel)
+# author: Bob Droege (@bedroge)
+# author: Hafsa Naeem (@hafsa-naeem)
+# author: Thomas Roeblitz (@trz42)
+#
+# license: GPLv2
+#
+import re
+
+
+def get_comment(pr, job_id):
```

Comment:
I'd suggest to replace `job_id` with `search_pattern` as a parameter to the function. Also see comment about lines 23 & 24 below.

Also docstring for the function would be good.